vm/cuttlefish/cuttlefish.go

```diff
@@ -81,11 +81,15 @@ func (pool *Pool) Create(workdir string, index int) (vmimpl.Instance, error) {
 		return nil, fmt.Errorf("failed to get root access to device: %s", err)
 	}
 
-	if err := inst.runOnHost(5*time.Minute, fmt.Sprintf("adb shell setprop persist.dbg.keep_debugfs_mounted 1;"+
-		"mount -t debugfs debugfs /sys/kernel/debug; chmod 0755 /sys/kernel/debug")); err != nil {
+	if err := inst.runOnHost(5*time.Minute, fmt.Sprintf("adb shell 'setprop persist.dbg.keep_debugfs_mounted 1;"+
+		"mount -t debugfs debugfs /sys/kernel/debug; chmod 0755 /sys/kernel/debug'")); err != nil {
 		return nil, fmt.Errorf("failed to mount debugfs to /sys/kernel/debug: %s", err)
 	}
 
+	if err := inst.runOnHost(5*time.Minute, fmt.Sprintf("adb shell mkdir %s", deviceRoot)); err != nil {
```

Comment:
Maybe also merge into the previous adb shell invocation?
I would also structure it as 1 command per line, long the lines of:
```
if err := inst.runOnHost(5*time.Minute, fmt.Sprintf("adb shell '"+
		"setprop persist.dbg.keep_debugfs_mounted 1;"+
		"mount -t debugfs debugfs /sys/kernel/debug;"+
		"chmod 0755 /sys/kernel/debug;"+
		"mkdir %s;"+
	'", deviceRoot)); err != nil {
	return nil, fmt.Errorf("failed to mount debugfs to /sys/kernel/debug: %s", err)
}
```

Reply:
Sounds good, thanks! I'll do that